src/System.CommandLine/OptionExtensions.cs

```diff
@@ -27,41 +27,58 @@ public static class OptionExtensions
             where TOption : Option
         {
             option.Argument.AddAllowedValues(values);
-            option.Argument.Suggestions.Add(values);
+            option.Argument.Completions.Add(values);
 
             return option;
         }
 
         /// <summary>
-        /// Adds suggestions for an option.
+        /// Adds completions for an option.
         /// </summary>
         /// <typeparam name="TOption">The type of the <see cref="Option" />.</typeparam>
-        /// <param name="option">The option for which to add suggestions.</param>
-        /// <param name="values">The suggestions to add.</param>
+        /// <param name="option">The option for which to add completions.</param>
+        /// <param name="values">The completions to add.</param>
         /// <returns>The option being extended.</returns>
-        public static TOption AddSuggestions<TOption>(
+        public static TOption AddCompletions<TOption>(
             this TOption option,
             params string[] values)
```

Comment:
should this be `CompletionItem`?

Reply:
This overload is already in use and is lower-ceremony for the common case where there's no supplemental information to be added. 

I do plan to populate the details field using symbol descriptions as part of this PR. So far, still just refactoring.